### PR TITLE
Refactor to Class

### PR DESCRIPTION
**Pull request by FranAmorim**

### CODE REVIEW

File: src/utils/score.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-Score.prototype.currentScore = function () {
-  return this.score
-}
+  addPoint () {
+    this.score += 1
+  }
 
-Score.prototype.addPoint = function () {
-  this.score += 1
+  get currentScore () {
```
Comment (AndreVarandas) [inline src/utils/score.js]: Can you move this before addPoint? 😃

File: src/utils/score.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
-function Score () {
-  this.score = null
-}
+export default class Score {
```
Comment (AndreVarandas) [inline src/utils/score.js]: Nope. Export default is evil.

Comment (FranAmorim) [reply src/utils/score.js]: Since its a class and the file is only for that i dont see any inconvenience in it.

File: src/utils/score.js (hunk 1 above)
Comment (FranAmorim) [reply src/utils/score.js]: Sure

File: src/utils/score.js (hunk 2 above)
Comment (AndreVarandas) [reply src/utils/score.js]: Heres something worth reading about default exports: https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html

**Pull request merged**